### DISCUSSION TITLE
How to create Non-human Primate head models

### DISCUSSION THREAD

**Discussion by liangsihan** [simnibs/simnibs#231] [7.Creating head models] [answered]
Hi,
May I ask if simnibs is capable of processing monkey(rhesus macaque) MRI scans and create head models accordingly?

**Comment by oulap** [comment, marked as answer]
Simnibs can not create the segmentation of a rhesus head for you from the MRI, but if you have such a segmentation you can use simnibs to create a mesh out of it, see here:

https://simnibs.github.io/simnibs/build/html/tutorial/advanced/custom_meshing.html

Also please make sure that the tissue labels match the standard simnibs tissue labels, i.e., WM is 1, GM is 2, CSF is 3, BONE is 4, and SCALP is 5.

For getting the segmentation there are some toolboxes around, for example this one:

https://brainvisa.info/axon-5.1/en/processes/categories/primatologist/category_documentation.html

but I'm not sure any of those segment non-brain tissues.

**Reply by liangsihan** [reply]
thanks a lot! I will try that
